zthin-parts/zthin/bin/refresh_bootmap

```
@@ -91,6 +91,30 @@ function cleanup_fcpdevices {
     done
```

Pls do the undedicate in the for fcp loop.
if wwpns is not null, do the original disconnectFCP in the wwpns loop.
else: call smcli to undedicate fcp. (remember to ignore errors of this step)